src/unix/tcp.c

```diff
@@ -445,9 +445,9 @@ int uv__tcp_listen(uv_tcp_t* tcp, int backlog, uv_connection_cb cb) {
 
   /* Start listening for connections. */
   tcp->io_watcher.cb = uv__server_io;
-  uv__io_start(tcp->loop, &tcp->io_watcher, POLLIN);
+  err = uv__io_start(tcp->loop, &tcp->io_watcher, POLLIN);
 
-  return 0;
+  return err;
```

Comment:
I know it's nitpicking but `return uv__io_start(tcp->loop, &tcp->io_watcher, POLLIN);` is IMO easier to read. Can't post as a suggestion.

Reply:
Nitpicking in C is pretty much a must. :) No worries at all.